src/RoadRegistry.BackOffice/Core/RoadNetwork.cs

```diff
@@ -71,9 +71,19 @@ private RoadNetwork(IRoadNetworkView view)
 
             var verifiedChanges = verifiableChanges.ConvertAll(change => change.AsVerifiedChange());
 
-            if (verifiedChanges.Count == 0) return;
-
-            if (verifiedChanges.OfType<RejectedChange>().Any())
+            if (verifiedChanges.Count == 0)
+            {
+                Apply(new Messages.NoRoadNetworkChanges
+                {
+                    RequestId = requestId,
+                    Reason = reason,
+                    Operator = @operator,
+                    OrganizationId = organization.Identifier,
+                    Organization = organization.Name,
+                    TransactionId = requestedChanges.TransactionId,
+                });
```

Comment:
Kan het zijn dat hier een `return` statement ontbreekt?
Anders gaan we zowel `NoRoadNetworkChanges` als `RoadNetworkChangesAccepted` applyen.

Reply:
Als hij in de `if` komt zal hij toch nooit in de `else` komen?

Reply:
Inderdaad, interpretatiefoutje van mij.